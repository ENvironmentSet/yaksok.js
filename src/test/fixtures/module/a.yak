'a' 보여주기
@b b 출력