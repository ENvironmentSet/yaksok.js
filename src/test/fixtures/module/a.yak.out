부수효과
a
b
